src/main/java/org/opentripplanner/model/OtpTransitService.java

```diff
@@ -18,9 +19,17 @@
 
     Collection<FeedInfo> getAllFeedInfos();
 
+    /**
+     * This is equivalent to a Transmodel Notice Assignments. The map key may reference entities of
+     * any type (having a {@link FeedScopedId} id).
+     */
+    Multimap<NoticeAssignable, Notice> getNoticeAssignments();
+
```

Comment:
Comment is wrong, the reason way we need to use `NoticeAssignable`and not element id is that StopTimes do not have ids. I am not sure what is best - adding ids to stopTimes or doing this. Adding ids would increase mem usage, while this rely on the eq./hashCode of the all classes being `NoticeAssignable` .